strawberryfields/backends/__init__.py

```diff
@@ -14,25 +14,36 @@
 """This module loads the required backend classes"""
 
 from .base import BaseBackend, BaseFock, BaseGaussian, ModeMap
-from .tfbackend import TFBackend
 from .gaussianbackend import GaussianBackend
 from .fockbackend import FockBackend
 
-__all__ = ['BaseBackend', 'BaseFock', 'BaseGaussian', 'FockBackend', 'GaussianBackend', 'TFBackend']
+__all__ = [
+    "BaseBackend",
+    "BaseFock",
+    "BaseGaussian",
+    "FockBackend",
+    "GaussianBackend",
+    "TFBackend",
+]
+
+supported_backends = {"base": BaseBackend, "gaussian": GaussianBackend, "fock": FockBackend}
 
-supported_backends = {"base": BaseBackend,
-                      "tf": TFBackend,
-                      "gaussian": GaussianBackend,
-                      "fock": FockBackend}
 
 def load_backend(name):
     """Loads the specified backend by mapping a string
     to the backend type, via the ``supported_backends``
     dictionary. Note that this function is used by the
     frontend only, and should not be user-facing.
     """
+    if name == "tf":
+        # treat the tensorflow backend differently, to
+        # isolate the import of tensorflow
+        from .tfbackend import TFBackend
```

Comment:
no try/except statement in case `TFBackend` fails to import?

Reply:
The try-except occurs during `TFBackend/__init__.py` import (or at least, it should)